src/CodeInvokerInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Dhii\Invocation;
+
+use Dhii\Util\String\StringableInterface as Stringable;
+
+/**
+ * Something that can invoke commands by code.
+ *
+ * @since [*next-version*]
+ */
+interface CodeInvokerInterface
```

Comment:
The name `CodeInvokerInterface` is a bit misleading, mainly due to the use of the term "code" which is ambiguous (program code vs key-like code). Consider using the term "command".